### ISSUE TITLE
[Screen Reader – Make Code - Home page] Footer landmark is not defined under ‘Home page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3433]
**User Experience**:
If the landmarks is not defined, then it will hinder the visually impacted users in navigating the application and also make difficulties to understand the application.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/# 
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify that footer landmark is defined on home page or not.

**Actual Result:**
Footer landmark is not defined under ‘Home page’.

**Expected Result:**
Footer landmark should be define under ‘Home page’.

**Note:**
Same issue exists under Chrome & Firefox.

**MAS Reference:**
[MAS 2.4.1 - Bypass Blocks](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d})

**Reference Links:**
•	[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
•	External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
•	Please reach out to [C&AI Teams channel](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A3cd071821b694f11b054d878453cd75f%40thread.skype%2Fconversations%3FgroupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&type=team&deeplinkId=99c73412-3928-459b-a051-ad90788e6969&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true) for any process related queries.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA
#URL:https://makecode.microbit.org/#

**Comment by Shalini3457** [comment]
![MAS2 4 1_Footer landmark is not defined](https://user-images.githubusercontent.com/72871089/96092971-2e300580-0ee9-11eb-92fc-b343971b7eb4.jpg)
